ts/components/select/types.ts

```diff
@@ -24,7 +24,8 @@ export type AsyncDataConfig = {
   api: string | null
   method: string
   params: any
-  alwaysFetch: boolean
+  alwaysFetch: boolean,
+  credentials?: 'include' | 'omit' | 'same-origin'
```

Comment:
The `Request.credentials` already have a ts type
```suggestion
  credentials?: RequestCredentials
```